sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/SharedGalleryImageTests.java

```diff
@@ -308,6 +309,7 @@ public void canCreateUpdateGetDeleteGalleryImageVersion() {
     }
 
     @Test
+    @Disabled("Creating a managed image with managedDisk source that has 'TrustedLaunch' security type is not supported.")
```

Comment:
Seems service has ended its support for creating trusted launch VMs using managed image:
https://learn.microsoft.com/azure/virtual-machines/capture-image-resource

created issue to track: https://github.com/Azure/azure-sdk-for-java/issues/42339